### PR TITLE
Add glibc-langpack-en

### PR DESCRIPTION
**Pull request by shanemcd**

### CODE REVIEW

**Review by AlanCoding — APPROVED** [review]

File: bindep_combined.txt [hunk 1]
```diff
@@ -16,3 +16,4 @@ qemu-img [platform:rpm]  # from collection ovirt.ovirt
 python3-devel [platform:rpm compile]  # from collection user
 subversion [platform:rpm]  # from collection user
 subversion [platform:dpkg]  # from collection user
+glibc-langpack-en  # from collection user
```
Comment (AlanCoding) [inline bindep_combined.txt]: would it be better to mark this with `[platform:rpm]`? I would look to @pabelanger for the answer

Comment (pabelanger) [reply bindep_combined.txt]: yup, you are right. Will fix.

**Review by pabelanger — APPROVED** [review]

**Comment by AlanCoding** [comment]
How do we get Zuul to push a new image to https://quay.io/repository/ansible/awx-ee?tag=latest&tab=tags? Or has it just not finished yet?

**Comment by pabelanger** [comment]
> How do we get Zuul to push a new image to [quay.io/repository/ansible/awx-ee?tag=latest&tab=tags](https://quay.io/repository/ansible/awx-ee?tag=latest&tab=tags)? Or has it just not finished yet?

yah, it is not finished yet. I am hoping to fix the delay on this, but creating a promote job to use the image we build in gate, over re-building it in post pipeline.

You can see job running in https://dashboard.zuul.ansible.com/t/ansible/status

**Pull request merged**